Dockerfile

```diff
@@ -255,7 +256,7 @@ RUN patchelf --add-needed libm-2.23.so /opt/deps/libvarnishapi.so.1.0.4
 ###### Python Plugin Image ######
 FROM collectd as python-plugins
 
-RUN python3 -m pip install yq &&\
+RUN python3 -m pip install --upgrade pip==$PIP_VERSION && python3 -m pip install yq &&\
```

Comment:
Need to also add `ARG PIP_VERSION` before this line for multi-stage builds.

Reply:
Thanks for spotting that 👍 didn't know it worked that way